### DISCUSSION TITLE
A little question

### DISCUSSION THREAD

**Discussion by BrStoker** [def-studio/telegraph#662] [Q&A] [answered]
Hi everyone. I send reply button to bot
```
Telegraph::chat($this->chat->chat_id)->markdown($message)
                ->replyKeyboard(ReplyKeyboard::make()->buttons([
                    ReplyButton::make('✅ Регистрация')->requestContact()
                ])->resize()->oneTime())->send();
```
How can I know that a user has sent me their contact?

**Comment by fabio-ivona** [comment, marked as answer]
Hi!

you should set up a webhook (docs [here](https://docs.defstudio.it/telegraph/v1/webhooks/overview)) or listen for updates on your bot (docs [here](https://docs.defstudio.it/telegraph/v1/api/bots#content-botupdates))

**Reply by BrStoker** [reply]
Reading the documentation was very helpful. I should have looked at it more carefully. Thank you very much.
